UnitsNet.Tests/CustomCode/PowerTests.cs

```diff
@@ -127,5 +127,19 @@ public void PowerDividedBySpecificEnergyEqualsMassFlow()
             MassFlow massFlow = Power.FromWatts(15.0) / SpecificEnergy.FromJoulesPerKilogram(3);
             Assert.Equal(massFlow, MassFlow.FromKilogramsPerSecond(5));
         }
+
+        [Fact]
+        public void CurrentMultipliedByPotentialEqualsPower()
+        {
+            ElectricCurrent i = ElectricCurrent.FromAmperes(5);
+            ElectricPotential u = ElectricPotential.FromVolts(10);
+            Power p = Power.FromWatts(50);
+            Assert.Equal(p, i * u);
+            Assert.Equal(p, u * i);
```

Comment:
These two test casees are already covered by the above two U\*I and I\*U tests. I think they can be removed here.

Reply:
See below